### PR TITLE
middleware.Compress: implement Unwrap() method to the compressResponseWriter

### PR DESCRIPTION
**Pull request by kei2100**
This PR solves https://github.com/go-chi/chi/issues/818

### CODE REVIEW

**Review by VojtechVitek — APPROVED** [review]
LGTM

**Comment by kei2100** [comment]
@VojtechVitek 
Thank you for the approval!

@pkieltyka 
Please let me know if there is any work I should do for the merge.

File: middleware/compress.go [hunk 1]
```diff
@@ -382,6 +382,10 @@ func (cw *compressResponseWriter) Close() error {
 	return errors.New("chi/middleware: io.WriteCloser is unavailable on the writer")
 }
 
+func (cw *compressResponseWriter) Unwrap() http.ResponseWriter {
+	return cw.ResponseWriter
```
Comment (VojtechVitek) [inline middleware/compress.go:386]: This is the same pattern as in https://github.com/go-chi/chi/blob/77d709fcde559ca625d4e126478835a0e85d79a9/middleware/wrap_writer.go#L116.

Going to merge.

**Review by VojtechVitek — APPROVED** [review]
LGTM

**Pull request merged**